run_dir/static/js/projects.js

```diff
@@ -498,6 +516,47 @@ $('body').on('click hidden.bs.dropdown', '.rBtngp2', function(event){
   read_current_filtering();
 });
 
+//Saving the filtering/sorting to user-defined preset
+$(document).on('click', '#saveFilter', function() {
+  var presetFilterName = '';
+  if ($('#user_presets_dropdown .dropdown-toggle').hasClass('active')){
+    //if user-defined preset selected, save to old preset
+    presetFilterName=$('#user_presets_dropdown .dropdown-toggle').text();
+    if (sort_preset !== presetFilterName){
+      //if selected preset does not match the loaded preset
+      alert('Click "Load Table" to save to Preset');
```

Comment:
Change message so that it describes why it should be done.

Reply:
Hmm, this kinda implies that the filter will be saved if the user clicks `Load Table`. I think the message should say that the preset does not match the table, to reload and that the filtering should be redone after the table is reloaded.